homeassistant/components/samsungtv/config_flow.py

```diff
@@ -96,81 +152,93 @@ async def async_step_import(self, user_input=None):
     async def async_step_user(self, user_input=None):
         """Handle a flow initialized by the user."""
         if user_input is not None:
-            ip_address = await self.hass.async_add_executor_job(
-                _get_ip, user_input[CONF_HOST]
+            self._host = await self.hass.async_add_executor_job(
```

Comment:
If we cannot resolve the hostname to an ip, we need to error. It may be easier to do this when we trigger import instead.

Reply:
I added the error here, as `async_step_import` just call this func